cmd/krel/cmd/release_notes.go

```diff
@@ -80,21 +84,28 @@ permissions to your fork of k/sig-release and k-sigs/release-notes.`,
 		github.TokenEnvKey),
 	SilenceUsage:  true,
 	SilenceErrors: true,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		// If none of the operation modes is defined, show the usage help and exit
+		if !releaseNotesOpts.createDraftPR && !releaseNotesOpts.createWebsitePR {
+			if err := cmd.Help(); err != nil {
+				logrus.Error(err)
```

Comment:
Would this make more sense in terms of error handling?
```suggestion
				logrus.Fatal(err)
```
Otherwise we could use `PreRunE` and return the error.